### DISCUSSION TITLE
Can you please remove this one?

### DISCUSSION THREAD

**Discussion by cednore** [rustdesk/rustdesk#9858] [General]
Rustdesk is great!
I chose that over AnyDesk, Google Remote Desktop.
Thanks for your continuous effort.
My only suggestion. can you please remove this one?
![image](https://github.com/user-attachments/assets/9d6421e5-42d8-470b-aa6d-803702b8edd4)
This one is really annoying when sharing my remote screen.
It would be really great if I try this new experience in demo next week.

Thanks,
Best regards,
Jason.

**Comment by Poordeveloper** [comment]
Ridiculous requirement. But you are lucky, this is open source, you can modify it if you do not like anything.
